src/codal_port/Makefile

```diff
@@ -45,7 +45,8 @@ INC += -I$(BUILD)
 CWARN = -Wall -Werror
 CWARN += -Wpointer-arith -Wuninitialized
 CFLAGS_ARCH += -DNRF52833_XXAA
-CFLAGS = $(INC) $(CWARN) -std=c99 $(CFLAGS_MOD) $(CFLAGS_ARCH) $(COPT) $(CFLAGS_EXTRA)
+CFLAGS += $(INC) $(CWARN) -std=c99 $(CFLAGS_MOD) $(CFLAGS_ARCH) $(COPT) $(CFLAGS_EXTRA)
+CXXFLAGS += $(filter-out -std=c99,$(CFLAGS))
```

Comment:
I removed a space between `,` and `$(CFLAGS)`, because it's not needed and space means something in Makefile's.